developers/weaviate/tutorials/connect.mdx

```diff
@@ -9,92 +9,95 @@ import WCSWithoutAuthentication from '/_includes/code/wcs.without.authentication
 import WCSAuthenticationApiKeyWithInferenceKey from '/_includes/code/wcs.authentication.api.key.with.inference.key.mdx';
+import EmbeddedInstantiationSimple from '/_includes/code/embedded.instantiate.simple.mdx';
 
-## Overview
+Learn how to connect to Weaviate.
 
-Here, you will learn how to connect to your instance of Weaviate, including how to provide authentication parameters such as the Weaviate API key, and inference API keys (e.g. Cohere, Hugging Face or OpenAI API keys).
+Configure your connection to provide authentication parameters and API keys for Weaviate and third-party API services like Cohere, Hugging Face and OpenAI.
 
 ## Prerequisites
 
-For this tutorial, you need to know:
-- The location of your Weaviate instance, and
-- The **configuration** of the Weaviate instance, especially:
-    - Whether **authentication is enabled**, and if so, the allowed authentication methods.
-    - If any **modules** are being used that use external inference APIs (e.g. OpenAI).
+To configure a connection, you need the following information:
 
-import EmbeddedInstantiationSimple from '/_includes/code/embedded.instantiate.simple.mdx';
+- The location of your Weaviate instance
+- If authentication is enabled, the allowed authentication methods
+- If you are using modules, the API keys and module connection details
 
-### For WCS users
+### Where to find connection details
 
-You can find the cluster URL, authentication status and enabled modules for each instance, in the Weaviate Cloud Services dashboard.
+**Weaviate Cloud Services (WCS)** If you are using Weaviate Cloud Services (WCS), the dashboard has a "Details" button. Click the button to get the following information:
 
-### For Docker/Kubernetes deployments
+ - The cluster URL
+ - Authentication type
+ - API keys, if enabled
+ - Module details
 
-You can find the relevant configuration in the configuration files, such as `docker-compose.yml` for Docker Compose, or `values.yaml` in the Helm chart.
+**Docker and Kubernetes** If you are using Docker or Kubernetes, your container configuration files have the connection details. 
 
-For Docker instances, your URL will likely be `http://localhost:8080` unless specified otherwise.
+- For Docker Compose, see `docker-compose.yml`.
+- For Kubernetes, see `values.yaml` in the Helm chart.
 
-:::tip For Embedded Weaviate
-If you are using Embedded Weaviate, you do not need to specify a URL at the client instantiation step, and most likely will not need any authentication.
-<br/>
-<p>
-So, you can simply instantiate with the following.
-</p>
+Docker instances often use the `http://localhost:8080` address.
 
-<EmbeddedInstantiationSimple/>
+**Embedded Weaviate** If you are using Embedded Weaviate, you do not need to specify a URL when you create the client. Embedded instances usually don't need authentication details. For details, see the [Embedded Weaviate documentation](../installation/embedded.md).
```

Comment:
How about changing `Embedded instances usually don't need authentication details.` to `Embedded instances by default do not require authentication credentials.`